controller/core/src/accounts/rebalance_record.rs

```diff
@@ -0,0 +1,58 @@
+use core::mem::size_of;
+
+use crate::internal_utils::{impl_cast_from_acc_data, impl_cast_to_acc_data};
+
+#[repr(C)]
+#[derive(Clone, Copy, Debug, PartialEq, Eq, Hash)]
+pub struct RebalanceRecord {
+    pub old_total_sol_value: u64,
+    pub dst_lst_index: u32,
```

Comment:
to keep consistent with the rest of the repo. Yes, what used to be `src, dst` in `S` is now `out, inp`
```suggestion
    pub inp_lst_index: u32,
```